bundles/org.eclipse.ui.browser/plugin.xml

```diff
@@ -216,12 +216,19 @@
          factoryclass="org.eclipse.ui.internal.browser.macosx.SafariBrowserFactory">
          <location>Applications/Safari.app</location>
       </browser>
-      <browser
+<!--  <browser
```

Comment:
same here: pls. delete.
In addition. Also delete browserInternetExplorer and browserCamino in plugin.propertie

Reply:
@BeckerWdf Deleted both camino and Internet Explorer.